components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+require('./Dropdown.scss')
+const React    = require('react')
+
+const Dropdown = {
```

Comment:
ES6 class for components with state (see SearchBar comments https://github.com/appirio-tech/react-components/commit/3118fed8f68e8ca4a511142d2cbdf982f9344a6d)